script/remote_deploy

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This script is copied to the deploy target by the deploy script and run there.
+# You probably don't want to run it directly.
+
+set -ex
+
+DOCKER_REPO=$(echo "$DOCKER_IMAGE" | cut -d: -f1)
+
+old_container_ids=$(docker ps | grep $DOCKER_REPO | awk '{print $1}')
+
+docker pull $DOCKER_REPO
+
+if [[ -n "$old_container_ids" ]]; then
+  for cid in "$old_container_ids"; do
+    docker stop $cid
+  done
+fi
+
+docker run -d -P -p 514:514/udp -e PAPERTRAIL_PORT=$PAPERTRAIL_PORT --name papertrail $DOCKER_IMAGE
```

Comment:
Since we're naming the container "papertrail" we will have to not only stop but delete the old one first or this will fail. This is about to change radically with CoreOS, but for now we'll need to `docker rm $cid` above around line 17.